sample_code/ep170/rev03/t.py

```diff
@@ -0,0 +1,32 @@
+import subprocess
+from typing import Final
+from typing import Literal
+from typing import overload
+from typing import Union
+
+
+@overload
+def output(*cmd: str, text: Literal[True]) -> str: ...
+@overload
+def output(*cmd: str, text: Literal[False] = False) -> bytes: ...
+
+
+def output(*cmd: str, text: bool = False) -> Union[bytes, str]:
+    return subprocess.check_output(cmd, text=text)
+
+
+# Union[Literal['passed'], Literal['failed'], Literal['errored']]
+# Literal['passed', 'failed', 'errored']
+
+PASSED: Literal['passed'] = 'passed'
+FAILED: Final = 'failed'
```

Comment:
`Final` example changed to 'failed' in order to make `mypy` happy, when running it with the example file.